Funky-Chickens/bangazonAPI
# Cait get order detail

I made the following changes:
    created the get all orders and get one order functionality in the model, controller and route files so developers have access to see and post to the orders table. 
    updated the order model to join the products table to single order get in order to show products on 

Steps to test:
go to http://localhost:8080/bangazonAPI/v1/orders to see if a list of orders appear
go to http://localhost:8080/bangazonAPI/v1/orders/"id#" to see if the details for that order appear with product details.
got to postman and run a post for the orders table and then get to see that added order is there
